charts/karmada/values.yaml

```diff
@@ -285,6 +285,10 @@ controllerManager:
     rollingUpdate:
       maxUnavailable: 0
       maxSurge: 50%
+  ## @param featureGate to controllerManager
+  featureGates:
+    ## @param PropagateDeps is a feature gate for the controllerManager to allow propagate dependent respurce to workloads.
+    PropagateDeps: false
```

Comment:
I am thinking that propagateDeps=false is a reasonable default value. How about setting the featureGates to empty by default?

Reply:
Empty is same effect with false by default. I think it's better to set a exact value to controller manager. 
the user can yet know the current value whart is.

Reply:
It seems make sense.